avocado/utils/archive.py

```diff
@@ -175,8 +175,8 @@ def _update_zip_extra_attrs(self, dst_dir):
         for path, info in self._engine.NameToInfo.iteritems():
             dst = os.path.join(dst_dir, path)
             if not os.path.exists(dst):
-                LOG.warn("Paths in this zip file are stored in unsupported "
-                         "format, not updating the attributes. (%s)", msg)
+                LOG.warn("The zip file '%s' does not exist, "
+                         "no way to update the attributes.", dst)
```

Comment:
Oh, now I understand the message you proposed.  IMO, a mix between the original message and yours would be the best choice.  The reason is that other paths will be skipped (see the `return` statement), and the reason for the file not existing is indeed because they're store in an unsupported format.  Proposal:

```python
LOG.warn("One or more files in the ZIP archive "%s" could not be found after extraction.  "
         "Their paths are probably stored in unsupported format and their attributes are" 
         "not going to be updated", self.filename)
```